pysph/examples/sloshing_tank_horizontal.py

```diff
@@ -48,16 +47,22 @@
 beta = 0.0
 gamma = 7.0
 
+length = 1.73
+h_tank = 1.15
+h_liquid = 0.6
+
 
 class HorizontalExcitation(Equation):
     def __init__(self, dest, sources):
+        self.amp = 0.032
+        self.T = 1.3
```

Comment:
Pass these variables to the `__init__` method so it can be configured from the equation.